Sources/eips/eip1024/EthEncryptedData+EIP1024.swift

```diff
@@ -51,12 +95,72 @@ extension EthEncryptedData {
         
         return message
     }
+    
+    
+    /// Decrypt the message using the sender's private key
```

Comment:
@ronaldmannak this looks good to me, but indentations needs to be fixed

Reply:
Done and done